cheatsheets/vimbox.rb

```diff
@@ -0,0 +1,208 @@
+cheatsheet do
+  title 'VimBox'               # Will be displayed by Dash in the docset list
+  docset_file_name 'vimbox'    # Used for the filename of the docset
+  keyword 'vb'             # Used as the initial search keyword (listed in Preferences > Docsets)
+  # resources 'resources_dir'  # An optional resources folder which can contain images or anything else
+
+  introduction 'Modern MacVim Configuration: Mac Keyboard mappings, autocomplete, bundles etc.'  # Optional, can contain Markdown or HTML
+
+  # A cheat sheet must consist of categories
+  category do
+    id 'Mac Key Commands'  # Must be unique and is used as title of the category
+
+    entry do
+      command 'CMD+P'         # Optional
+      command 'CTRL+P'        # Multiple commands are supported
+      name 'Open Anything and begin searching for file'    # A short name, can contain Markdown or HTML
+    end
+    entry do
+      command 'CMD+N'
+      name 'New Tab'
```

Comment:
Well, the trick is, that Vimbox's feature is exactly to setup these common actions, which are not there normally. That's why I added them.